tests/util.py

```diff
@@ -40,9 +40,12 @@
 )
 from spandrel.__helpers.model_descriptor import StateDict
 from spandrel.util import KeyCondition
-from spandrel_extra_arches import EXTRA_REGISTRY
 
-MAIN_REGISTRY.add(*EXTRA_REGISTRY)
+# The asserts check that the install function first does return
+# the newly installed architectures and then does not return them
+# when requested to ignore duplicates.
+assert spandrel_extra_arches.install()
+assert not spandrel_extra_arches.install(ignore_duplicates=True)
```

Comment:
Having functions with side effects in [`assert`](https://docs.python.org/3/reference/simple_stmts.html#assert) seems like a bad idea to me.

```suggestion
# The asserts check that the install function first does return
# the newly installed architectures and then does not return them
# when requested to ignore duplicates.
_installed_extras = spandrel_extra_arches.install()
assert len(_installed_extras) > 0
assert len(spandrel_extra_arches.install(ignore_duplicates=True)) == 0
```